contracts/src/TeleporterTokenDestination.sol

```diff
@@ -162,6 +162,8 @@ abstract contract TeleporterTokenDestination is
             messageType: BridgeMessageType.REGISTER_DESTINATION,
             payload: abi.encode(registerMessage)
         });
+
+        _handleFees(feeInfo.feeTokenAddress, feeInfo.amount);
```

Comment:
I think the return value of `_handleFees` is what needs to be used as the amount passed to `_sendTeleporterMessage`, right?

Reply:
yes that's correct